src/irc/__init__.py

```diff
@@ -130,6 +131,7 @@ def _get(data, key, format_found=None, default=None):
     if key in data:
         value = data[key]
         if value:
+            value = _sanitize(value)
```

Comment:
Can you move this to the outside again?

Reply:
...sigh no idea how that got back in there, #[21b5574](https://github.com/valcanobacon/BoostBots/pull/10/commits/21b5574adebf168ddd72f073c71b62fa4fe61d3a) fixes

Reply:
As to unittest, is that blocking merging this?